README.md

```diff
@@ -191,22 +191,41 @@ py.test
 #### The test data subset
 If you add new tables to the data, you'll need to generate a new subset for testing. We use this nifty subsetting tool: [rdbms-subsetter](https://github.com/18F/rdbms-subsetter).
 
-To build a new test subset, use the `build_test` invoke task:
+To build a new test subset, first create a new database for yourself:
+
+```
+createdb cfdm_new_subset
```

Comment:
Did you overwrite the current dump directly when doing this or your `cfdm_test` database?  We certainly could, I happened to run into an issue for some reason when doing so because some data/structures already existed (I can't remember the specific error), and overwriting the previous dump itself didn't work for me due to the need to supply a Postgres connection string as opposed to a file path.

If there's a way to do this without having to go through some of these extra steps, I'd much rather do that! :-)